### PR TITLE
Calendar Set Up

### PR DESCRIPTION
**Pull request by bellaboulais**
Set up the calendar page using react-big-calendar.

React-Big-Calendar Documentation: (https://www.npmjs.com/package//react-big-calendar). 
Implemented basic functionality and changed styles to match our app i.e. removed week/day options since we do not yet have time components, formatted due date component to work with the calendar start and end accessors, created a modal to display tasks when pressed on. 
Implemented react-navigate to link the calendar page to the sidebar.

### CODE REVIEW

File: packages/react-frontend/src/MyCalendar.js
Comment (iemxia) [inline packages/react-frontend/src/MyCalendar.js:1]: You also used a modal like me, it was cool to see a different implementation of it!

**Review by iemxia — APPROVED** [review]
Lots of progress, really useful Calendar view and side bar implementation. Well done, only thing I can think of is maybe more code comments to make it clearer what each section is doing (I am also guilty of not writing enough comments)

**Review by eckester — APPROVED** [review]
This all looks amazing, creating the calendar was one of the hardest things we needed to do, and I am so excited about how this turned out. Works great, I could not find any errors or bugs, at least when I went over it.

**Comment by bellaboulais** [comment]
> Lots of progress, really useful Calendar view and side bar implementation. Well done, only thing I can think of is maybe more code comments to make it clearer what each section is doing (I am also guilty of not writing enough comments)

I agree, I will definitely add comments to make it more readable. Comments are an easy thing to forget!

**Review by Crosa01 — APPROVED** [review]
I had a lot of fun playing around with the calendar view and seeing tasks displayed in it. It works amazingly, really good job. The code looks clean and does exactly what it was programmed to do!

**Pull request merged**